hazelcast/tests/docker/Dockerfile

```diff
@@ -1,10 +1,13 @@
 ARG HAZELCAST_VERSION
 FROM hazelcast/hazelcast:${HAZELCAST_VERSION}
 
+
+ARG LOG4J2_VERSION
+
 RUN apk update && \
     apk add curl tar && \
-    curl -LO https://downloads.apache.org/logging/log4j/2.13.1/apache-log4j-2.13.1-bin.tar.gz && \
-    tar -xzf apache-log4j-2.13.1-bin.tar.gz && \
+    curl -LO https://downloads.apache.org/logging/log4j/${LOG4J2_VERSION}/apache-log4j-${LOG4J2_VERSION}-bin.tar.gz && \
+    tar -xzf apache-log4j-${LOG4J2_VERSION}-bin.tar.gz && \
     mkdir /tmp/log4j2 && \
-    mv apache-log4j-2.13.1-bin/log4j-api-2.13.1.jar /tmp/log4j2/ && \
-    mv apache-log4j-2.13.1-bin/log4j-core-2.13.1.jar /tmp/log4j2/
+    mv apache-log4j-${LOG4J2_VERSION}-bin/log4j-api-${LOG4J2_VERSION}.jar /tmp/log4j2/ && \
+    mv apache-log4j-${LOG4J2_VERSION}-bin/log4j-core-${LOG4J2_VERSION}.jar /tmp/log4j2/
```

Comment:
I think it might be more reliable to download them directly from maven central. Since apache seems to remove old artefacts. 

Here are the equivalent links:

```
https://repo1.maven.org/maven2/org/apache/logging/log4j/log4j-core/2.13.2/log4j-core-2.13.2.jar
https://repo1.maven.org/maven2/org/apache/logging/log4j/log4j-api/2.13.2/log4j-api-2.13.2.jar
```

Reply:
Nice, thanks!!!